website/i18n/en.toml

```diff
@@ -1,3 +0,0 @@
-# Fix a typo in Docsy (although we don't currently use the print support)
```

Comment:
Was this fixed upstream?

Reply:
Yes, via this [PR](https://github.com/google/docsy/pull/760).